x-pack/plugins/triggers_actions_ui/server/plugin.ts

```diff
@@ -20,13 +28,16 @@ export class TriggersActionsPlugin implements Plugin<void, PluginStartContract>
     this.data = getService();
   }
 
-  public async setup(core: CoreSetup): Promise<void> {
+  public async setup(core: CoreSetup, plugins: PluginsSetup): Promise<void> {
+    const router = core.http.createRouter();
     registerDataService({
       logger: this.logger,
       data: this.data,
-      router: core.http.createRouter(),
+      router,
       baseRoute: '/api/triggers_actions_ui',
     });
+
+    healthRoute(this.logger, router, '/api/triggers_actions_ui', plugins.alerts !== undefined);
```

Comment:
We could store `/api/triggers_actions_ui` in a const since it is used more than once in this file.